sql-cli/sql_cli/cli/config.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import annotations
+
+import json
+from pathlib import Path
+
+import typer
+
+from sql_cli.astro.command import AstroCommand
+from sql_cli.cli.utils import resolve_project_dir
+from sql_cli.constants import DEFAULT_ENVIRONMENT
+
+app = typer.Typer()
+
+
+class InvalidConfigException(Exception):
+    pass
+
+
+def _get(key: str, project_dir: Path, env: str, as_json: bool) -> str:
+    """
+    Typer-agnostic implementation of the `config get` command.
+
+    :param key: Key to be fetched from the configuration file.
+    :param project_dir: Path to the project directory
+    :param env: SQL CLI environment (default, dev)
+    :param as_json: If the output should be displayed as JSON.
+    :returns: Either the string containing the key value or a JSON containing desired the key-value pair(s)
+    """
+    from sql_cli.configuration import Config
+
+    project_dir_absolute = resolve_project_dir(project_dir)
+    project_config = Config(environment=env, project_dir=project_dir_absolute).from_yaml_to_config()
+    if key:
+        return getattr(project_config, key)
+    elif as_json:
+        return json.dumps(project_config.to_dict())
+    else:
+        raise InvalidConfigException("Please, either define a key or use the --as-json flag")
+
+
+def _set(key: str, project_dir: Path, env: str, astro_deployment: str, astro_workspace: str) -> None:
+    """
+    Set deployment configuration associated to a SQL CLI environment.
+
+    :param key: Configuration property (key) to be set. At the mmoment only "deploy" is accepted.
+    :param project_dir: Path to the project directory.
+    :param env: SQL CLI environment (default, dev).
+    :param astro_deployment: Astronomer Cloud deployment ID.
```

Comment:
Should we rename this to `astro_deployment_id` ?

Reply:
:+1: I'm fine with it, @pankajkoti , it would make it more consistent with the Astro CLI
https://github.com/astronomer/docs/blob/main/astro/cli/astro-deploy.md